avm/res/network/dnsresolver/outbound-endpoint/main.bicep

```
@@ -0,0 +1,44 @@
+metadata name = 'DNS Resolver Outbound Endpoint'
+metadata description = 'This module deploys a DNS Resolver Outbound Endpoint.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Required. Name of the Private DNS Resolver.')
+@minLength(1)
+param dnsResolverName string
+
+@description('Optional. Location for all resources.')
+param location string = resourceGroup().location
+
+@description('Optional. Tags of the resource.')
+param tags object?
+
+@description('Required. The subnet ID of the inbound endpoint.')
+param subnetId string
```

```suggestion
@description('Required. The subnet resource ID of the inbound endpoint.')
param subnetResourceId string
```

Requires updating in references too